db_management.py

```diff
@@ -17,36 +17,45 @@ class User(Base):
     Password = Column(String, nullable=False, unique=True)
 
 
-def create_connection():
+def create_tables():
     Base.metadata.create_all(engine)
 
 
-def check_if_user_exists(str_username, str_pass):
+def create_connection():
+    engine = create_engine('sqlite:///data_base.db')
+    conn = engine.connect()
+    Session = sessionmaker(bind=engine)
+    Base = declarative_base()
+
+
+def check_user_exists(str_username: str, str_pass: str) -> bool:
     Session = sessionmaker(bind=engine)
     session = Session()
 
     user_count = session.query(User).filter\
         (User.Username == str_username and User.Password == str_pass).count()
-    if user_count == 0:
-        return False
-    return True
+    return user_count > 0
 
 
-def is_username_taken(str_input):
+def is_username_taken(username: str) -> bool:
     Session = sessionmaker(bind=engine)
     session = Session()
 
     user_count = session.query(User).filter \
-        (User.Username == str_input).count()
+        (User.Username == username).count()
     if user_count > 0:
```

Comment:
We said we can eliminate that condition and use a return instead, right? why didn't you change that?

Reply:
too many ctrl+z. changed it again